MANIFEST.in

```diff
@@ -3,3 +3,5 @@ include LICENSE
 include .coveragerc
 include .pylintrc
 recursive-include rules *.py
+exclude Dockerfile
+exclude .travis
```

Comment:
I also had to add `recursive-exclude .travis *` for that to work, as `exclude` is not recursive